File: uri/C/beginner/1019.c
#include <stdio.h>

int main ()
{
	int N, horas=0, minutos=0, segundos=0;

	scanf("%d", &N);

	if(N >= 3600){
		horas = N/3600;  
		N = N % 3600;
	}

	if(N >= 60){
		minutos = N/60;
		N = N % 60;
	}

	if(N < 60){
		segundos = N;
	}

	printf("%d:%d:%d\n", horas, minutos, segundos);

	return 0;
}

File: uri/python/1019.py
# -*- coding: utf-8 -*-

tempo = int(input())

horas = 0
minutos = 0
segundos = 0

while tempo > 0:
    if tempo >= 3600:
        horas = horas + 1
        tempo = tempo - 3600
    elif tempo < 3600 and tempo >= 60:
        minutos = minutos + 1
        tempo = tempo - 60
    else:
        segundos = tempo
        break 

print('{}:{}:{}'.format(horas, minutos, segundos))

